### ISSUE TITLE
php -  undefined method `sdk_path_if_needed' for OS::Mac:Module

### ISSUE THREAD

**Issue by jajalvipul** [Linuxbrew/homebrew-core#9879]
- [x] have a problem with `brew install` 
- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [x] ran `brew gist-logs php `  (https://gist.github.com/jajalvipul/890533c98b22882751834f358724e8f5) 
- [ ] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:
- What you were trying to do (and why)
I was trying to install the formula php using command "brew install php". 

- What happened (include command output)
Encountered the error shown below and unable to install formula "php"
```

[fsep@localhost Downloads]$ brew install php
==> Downloading https://php.net/get/php-7.2.11.tar.xz/from/this/mirror
Already downloaded: /home/fsep/.cache/Homebrew/downloads/5d0ee9fb096b97464d061a454967a5577a077b4c48d88d284ea8f0b390e1ebf8--php-7.2.11.tar.xz
==> Patching
patching file acinclude.m4
Hunk #1 succeeded at 444 (offset 3 lines).
Hunk #2 succeeded at 459 (offset 3 lines).
==> ./buildconf --force
Error: An exception occurred within a child process:
  NoMethodError: undefined method `sdk_path_if_needed' for OS::Mac:Module

```
- What you expected to happen
The expectation is that the installtion of formula "php" is successful.

- Step-by-step reproduction instructions (by running `brew install` commands)
Run the command 
```
brew install php
``` 

Also executed following workaround suggested for the similar issue relating undefined method but it did not resolve my issue.

```
brew update && brew upgrade brew-cask && brew cleanup && brew cask cleanup
```

Please advise to address the issue and install the formula "php" successfully. 
Thank you for your help.

**Comment by iMichka** [comment]
I am currently working on a patch: #10085

**Comment by iMichka** [comment]
Fix by #10085
